File: truth_set/python/documentation/textual/naming/legacy.py
def proc(u):
    if not u or 'nm' not in u:
        raise ValueError("Invalid input")
    u['a'] = u['nm'].upper()
    return u

user_data = {
    'nm': 'john doe',  
    'a': '' 
}

result = proc(user_data)
print(result)

File: truth_set/go/documentation/textual/naming/legacy.go
package naming

import (
    "fmt"
    "strings"
)

func proc(u map[string]string) map[string]string {
    if u == nil || u["nm"] == "" {
        panic("Invalid input")
    }

    u["a"] = strings.ToUpper(u["nm"])

    return u
}

func main() {
    userData := map[string]string{
        "nm": "john doe",
        "a":  "",
    }

    result := proc(userData)
    fmt.Println(result)
}
\ No newline at end of file